src/events/client/interactionCreate.ts

```diff
@@ -131,18 +132,51 @@ export default async function interactionCreateHandler(interaction: Eris.PingInt
         if (!gameSession.gameRound) return;
         await gameSession.guessSong(messageContext, guildPreference.gameOptions.guessModeType !== GuessModeType.ARTIST ? gameSession.gameRound.songName : gameSession.gameRound.artistName);
     } else if (interaction instanceof Eris.CommandInteraction) {
-        // Bookmarking songs
-        const gameSession = state.gameSessions[interaction.guildID];
-        const song = gameSession?.getSongFromMessageID(interaction.data.target_id);
-        if ((!gameSession || !song) && withinInteractionInterval(interaction)) {
-            await tryCreateErrorMessage(interaction, !gameSession ? "You can only bookmark songs during a game." : `You can only bookmark songs recently played in the last ${BOOKMARK_MESSAGE_SIZE} rounds. You must bookmark the message sent by the bot containing the song.`);
-            return;
-        }
+        if (interaction.data.type === Eris.Constants.ApplicationCommandTypes.USER) {
+            // Profile
+            const userID = interaction.data.target_id;
```

Comment:
Can we refactor this file to not include business logic? 
1. Move the multiple choice selection logic to game_session
2. Move this logic to `,profile`
3. Move bookmark logic to game_session  

Also, how do we differentiate between interactions of the same type, but different commands?

Reply:
> Also, how do we differentiate between interactions of the same type, but different commands?

`interaction.data.name`